src/includes/shape.js

```diff
@@ -161,26 +161,23 @@ pub.vertex = function() {
 };
 
 /**
- * The arc() function draws an arc in the display window.
- * Arcs are drawn along the outer edge of an ellipse defined by the
- * <b>x</b>, <b>y</b>, <b>width</b> and <b>height</b> parameters.
- * The origin or the arc's ellipse may be changed with the
- * <b>ellipseMode()</b> function.
- * The <b>start</b> and <b>stop</b> parameters specify the angles
- * at which to draw the arc.
+ * The arc() function draws an arc. Arcs are drawn along the outer edge of an ellipse
```

Comment:
Ok, I will take the HTML tags out for now, as they are not used throughout the docs consistently. We can later decide if we should use HTML tags for certain things, but if we do so, we should do it consistently and not in a few random functions.